context/src/bazel_bep/parser.rs

```diff
@@ -0,0 +1,99 @@
+use anyhow::Ok;
+use bazel_bep::types::build_event_stream::{build_event::Payload, file::File::Uri, BuildEvent};
+use serde_json::Deserializer;
+
+#[derive(Debug, Clone)]
+pub struct TestResult {
+    pub cached: bool,
+    pub xml_files: Vec<String>,
+}
```

Comment:
I was under the impression that one test result = one file - is that not the case? Or just being proactive here?

Reply:
From what I've seen it's just one xml file. But there are multiple files (test.log), and conceivably if you were using bazel to wrap some other invocation, you could get multiple junits, so being proactive